miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/ListBoxesController.java

```diff
@@ -25,8 +35,32 @@ public void setBoxService(BoxService boxService) {
   public String title() {
     return "Boxes";
   }
+
   @RequestMapping("/boxes")
   public ModelAndView listBoxes(ModelMap model) throws Exception {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)